MLP_Model/nbconverted/Hyperparameter_Optimization_model.py

```diff
@@ -277,16 +351,24 @@
     title="Loss vs. Epochs",
     x_axis_label="Epochs",
     y_axis_label="Loss",
+    params=params,
 )
 
 
-# In[15]:
+# In[18]:
 
 
 # calling the testing function and outputing list values of tested model
-y_pred_list = test_optimized_model(
-    model, test_loader, IN_FEATURES, OUT_FEATURES, param_dict, params
-)
+if params.MODEL_TYPE == "Multi_Class":
+    y_pred_list = test_optimized_model(model, test_loader, params)
+elif params.MODEL_TYPE == "Binary_Classification":
+    y_pred_list, y_pred_prob_list = test_optimized_model(model, test_loader, params)
+elif params.MODEL_TYPE == "Regression":
+    y_pred_list = test_optimized_model(model, test_loader, params)
```

Comment:
Consider simplifying here to account for the similarity of each if branch. For example, could you perform the same work for `Multi_Class` and `Regression` `MODEL_TYPE`'s?

Reply:
changed to simply